test/parallel/test-node-run.js

```diff
@@ -94,4 +101,16 @@ describe('node run [command]', () => {
     assert.strictEqual(child.stderr, '');
     assert.strictEqual(child.code, 0);
   });
+
+  it('will search subdirectories for a package.json file', async () => {
```

Comment:
My understanding is this PR is searching parent folders and not subdirectories. I think the name of the test is incorrect.

Reply:
Yes, parent-folders are correct. If this is not a blocking comment, I'd like to merge this, and follow up on a different PR.

Reply:
Updated the test message. Thanks!